documentation/src/main/docs/faq.adoc

```diff
@@ -9,13 +9,6 @@ To transform items synchronously use the `apply` operators:
 include::../../../src/test/java/snippets/HowToTransformTest.java[tags=sync]
 ----
 
-If your operation throws a _checked exception_, you can use the `io.smallrye.mutiny.unchecked.Unchecked` wrappers to avoid having to catch the exception in the _callback_.
```

Comment:
Did these checked exception sections mean to be removed?

Reply:
gosh... no.... Let me check what went wrong

Reply:
A rebase went terribly wrong. It's fixed now.